sql/plan/hash_lookup.go

```diff
@@ -64,15 +65,23 @@ func (n *HashLookup) WithExpressions(exprs ...sql.Expression) (sql.Node, error)
 
 func (n *HashLookup) String() string {
 	pr := sql.NewTreePrinter()
-	_ = pr.WriteNode("HashLookup(child: %v, lookup: %v)", n.childProjection, n.lookupProjection)
-	_ = pr.WriteChildren(n.UnaryNode.Child.String())
+	_ = pr.WriteNode("HashLookup")
+	children := make([]string, 3)
+	children[0] = fmt.Sprintf("source: %s", n.lookupProjection)
```

Comment:
Either the name of the field is bad or the debug output is

Think on your sins